tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -43,7 +43,7 @@ steps:
 
 - task: Bash@3
   name: SetVersion
-  displayName: Set Package Version
+  displayName: Calculate Package Version
```

Comment:
What is the display change for? There certainly seem to be tasks in here that are setting versions for something. Example: use of update-package-version.sh

Reply:
Didn't mean to include that change here. I find the "set" term confusing because we're not really setting the package versions in this step. That happens later in the "Update package versions" step. Regardless, though, that change doesn't belong here. I'll revert.

Reply:
```suggestion
  displayName: Set Package Version
```